pkg/appconsts/appconsts.go

```diff
@@ -72,16 +72,11 @@ const (
 	// data square.
 	MinShareCount = DefaultMinSquareSize * DefaultMinSquareSize
 
-	// SubtreeRootThreshold dictates the threashold for increasing the sub
-	// tree root height for blobs. If the number of subtree roots used to create
-	// a share commitment surpasses this threshold, then the height is
-	// increased. The rationale for this value is described in more detail in
-	// ADR013 (./docs/architecture/adr-013). Note that this value should not
-	// drop below the max square size. That could result in the caculating a sub
-	// tree width for a blob that is larger than whatever the actual suquare
-	// size for that block. Having a value that is larger than the square is
-	// simply wasteful in that the proofs for large blobs are unnecessarily
-	// large.
+	// SubtreeRootThreshold works as a target value for the number of subtree roots in the
+	// share commitment. If a blob contains more shares than this number, than the height
+	// of the subtree roots will gradually increases to so that the amount remains within that limit.
```

Comment:
```suggestion
	// share commitment. If a blob contains more shares than this number, then the height
	// of the subtree roots will increase so that the number of subtree roots remains below the threshold.
```